pkg/search/parser.go

```diff
@@ -84,6 +84,50 @@ func GetFieldValueFromQuery(query string, label FieldLabel) (string, bool) {
 	return "", false
 }
 
+func splitQuery(query string) []string {
```

Comment:
I think that it would be beneficial to simply test these functions distinctly. In fact, I think that parser_test.go already has this